planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -47,10 +47,10 @@
       {
         "SQL": "explain select /*+ broadcast_join(fact_t,d1_t) */ count(*) from fact_t, d1_t where fact_t.d1_k = d1_t.d1_k",
         "Plan": [
-          "StreamAgg_32 1.00 root  funcs:count(Column#14)->Column#11",
-          "└─TableReader_33 1.00 root  data:StreamAgg_13",
-          "  └─StreamAgg_13 1.00 cop[tiflash]  funcs:count(1)->Column#14",
-          "    └─BroadcastJoin_31 8.00 cop[tiflash]  inner join, left key:test.fact_t.d1_k, right key:test.d1_t.d1_k",
+          "HashAgg_24 1.00 root  funcs:count(Column#12)->Column#11",
```

Comment:
that's amazing, why the plan ID is reduced 😂

Reply:
Maybe because the whole plan is changed from `StreamAgg` to `HashAgg`?